lib/rubocop/cop/rails/pluralization_grammar.rb

```diff
@@ -0,0 +1,97 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for correct grammar when using active support's
+      # core extensions to the numeric classes
+      #
+      # @example
+      #   # bad
+      #   3.day.ago
+      #   1.months.ago
+      #
+      #   # good
+      #   3.days.ago
+      #   1.month.ago
+      class PluralizationGrammar < Cop
+        SINGULAR_DURATION_METHODS = { second: :seconds,
+                                      minute: :minutes,
+                                      hour: :hours,
+                                      day: :days,
+                                      week: :weeks,
+                                      fortnight: :fortnights,
+                                      month: :months,
+                                      year: :years }
+
+        PLURAL_DURATION_METHODS = SINGULAR_DURATION_METHODS.invert
+
+        MSG = 'Prefer `%s.%s` for grammar reasons.'
```

Comment:
I'd suggest `Prefer `%s.%s` over `%s.%s`.`. I believe further explanation here is redundant.